firestore-bigquery-export/functions/src/config.ts

```diff
@@ -35,6 +35,7 @@ export default {
   bqProjectId: process.env.BIGQUERY_PROJECT_ID,
   collectionPath: process.env.COLLECTION_PATH,
   datasetId: process.env.DATASET_ID,
+  docsPerBackfill: parseInt(process.env.DOCS_PER_BACKFILL),
```

Comment:
Did we test this with non-numeric values? What happens?

Reply:
The validation regex will reject any non-numeric values